docker-compose.yml

```diff
@@ -86,8 +86,15 @@ services:
     volumes:
       - orderly_volume:/orderly
     command: --port 8321 --go-signal /orderly_go /orderly
+  static:
+    image: ${MONTAGU_REGISTRY}/montagu-static:${MONTAGU_STATIC_VERSION}
+    restart: always
+    logging: *log-journald
+    volumes:
+      - static_volume:/www
```

Comment:
If we wanted to persist logs, we might add

```
      - static_logs:/var/log/caddy
```

and 
* add an extra volume below in `volumes:` 
* update the paths here https://github.com/vimc/montagu-static/blob/8c9e0e06a8808dd2831f85a2c716ffeaa43eabc1/caddy/Caddyfile#L6-L7 to add the `caddy` part to the path

Not sure how worthwhile this would be, but I think we increasingly do this on other bits of dockerised infrastructure